### ISSUE TITLE
沈黙ボタン

### ISSUE THREAD

**Issue by uhyo** [uhyo/jinrou#111]
投票時間ありのとき、全員が沈黙すると投票時間に移ることができる

**Comment by Sunev** [comment]
I think this button aims to something which is almost same as ```shorten the remaining time```, doesn't it?

**Comment by uhyo** [comment]
True. But I heard from some players that they want that functionality even when a game master is absent.
I may put this issue a low priority.
